src/pricecypher/dataclasses/models.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+
+type Models = dict[str, Model]
+
+@dataclass(frozen=True)
+class Model:
+    """Defines a (trained) model.
+
+    model (object): The trained model.
+    model_details (dict): A dict with model details.
+    """
+    model: object
+    model_details: dict
```

Comment:
A model consists of the model object, and some meta data (the details). Same as the current implementation in the generic datascience scripts.

Reply:
Replaced by the [contracts/pricecypher_model.py](https://github.com/marketredesign/pricecypher_python_sdk/blob/2e1123a99f087c892ce39ae07f0e2f54934e5a93/src/pricecypher/contracts/pricecypher_model.py) class